src/ANum.java

```diff
@@ -0,0 +1,37 @@
+import java.util.ArrayList;
+import java.util.LinkedHashSet;
+
+public class ANum {
+	
+
+
+	public static void getPairs() {
+		ArrayList<Integer> checked = new ArrayList<Integer>();
+		for (int i = 0; i < 1000000; i++) {
+			int a = sumAllFactors(i);
+			int b = sumAllFactors(a);
+			if (i == b && i != a) {
+				if (!checked.contains(i))
+					checked.add(i);
+				if (!checked.contains(b))
+					checked.add(b);
```

Comment:
small mistake: you should add `int a` instead of `int b` bc `b== i`. Here you should name the variables differently to add clarity and avoid this kind of mistake.